.travis.yml

```diff
@@ -7,9 +7,9 @@ env:
   matrix:
     - TEST=javascript
     # need node installs for app_manager.XpathValidatorTest
-    - TEST=python NOSE_DIVIDED_WE_RUN=05 JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=6a JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=bf JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=03 JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=69 JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=ba JS_SETUP=yes
```

Comment:
It also might make sense to add a 4th runner if 3 can no longer handle all the tests without timeout.